start_node.in

```diff
@@ -0,0 +1,5 @@
+#!@SHELL@
+
+set -e
+wget -O @START_NODE_DIR@/agent.jar https://ci.kronosnet.org/jnlpJars/agent.jar
+exec java -jar @START_NODE_DIR@/agent.jar
```

Comment:
before executing java, can we add:
rm -rf @START_NODE_DIR@/.jenkins ?

That dir contains the jenkins cache and it tends to grow substantially over time. It´s good to wipe it before firing up the new instance. That was the main driver to have the ability to update and add stuff to start_node.